src/todos/todos.service.ts

```diff
@@ -1,4 +1,40 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable } from "@nestjs/common";
+import { InjectModel } from "@nestjs/sequelize";
+import { CreateToDoDto } from "./dto/create-todo.dto";
+import { Todo } from "./todos.model";
 
 @Injectable()
-export class TodosService {}
+export class TodosService {
+  constructor(@InjectModel(Todo) private todoRepository: typeof Todo) {}
+
+  async createToDo(dto: CreateToDoDto) {
+    const todo = await this.todoRepository.create(dto);
+    return todo;
+  }
```

Comment:
You should rename the method, at other methods you're using Todos not ToDo

Reply:
Ok